### DISCUSSION TITLE
Direct Sampling

### DISCUSSION THREAD

**Discussion by matthcg** [AlexandreRouma/SDRPlusPlus#694] [General]
Apologies if I'm just missing it but I would like to listen to shortwave but for the life of me I can't find the direct sampling setting? Any help is appreciated. 

On a side note I find it really tough googling anything SDR++ related. Search results are always just generic RTL-SDR. Anyone else got the same problem?

**Comment by AlexandreRouma** [comment]
![image](https://user-images.githubusercontent.com/17126412/159006476-5d3a910a-336b-4392-a033-405b567513a4.png)

**Comment by matthcg** [comment]
I'm wondering if I got something disabled as it's not showing up. I'll
delete the configs and reinstall.

Compiled on Debian Bullseye from src.

--
Matt Gillard
C:6477749124

**Reply by AlexandreRouma** [reply]
it can't be hidden by anything config related so the only explanation is you either haven't selected the RTL-SDR source or you're running an extremely old version

**Reply by matthcg** [reply]
I'm using the SDRplay RSPdx. Doesn't look like it's a setting when using that device.

**Reply by AlexandreRouma** [reply]
Well which SDR you're using would have been useful to know...

The SDRplay doesn't need direct sampling. Install the API, make sure it's running and select the "SDRplay" option in the source menu

**Comment by matthcg** [comment]
I'm using the SDRplay RSPdx. Doesn't look like it's a setting when using
that device.

On Fri, Mar 18, 2022 at 9:42 AM AlexandreRouma ***@***.***>
wrote:

> it can't be hidden by anything config related so the only explanation is
> you either haven't selected the RTL-SDR source or you're running an
> extremely old version
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AlexandreRouma/SDRPlusPlus/discussions/694#discussioncomment-2387856>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYJICJKUWNIKXIZVAERNQHDVASB6BANCNFSM5RBYEYGA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Matt Gillard
C:6477749124

**Reply by AlexandreRouma** [reply]
As I've already responded on march 18, there is no such thing as direct sampling on the RSPs. it receives HF directly with no other setting necessary.

**Comment by matthcg** [comment]
Very odd. I did not post that message 3 hours ago. I'm unsubscribing from
this thread as the question was answered in full on March 18th.

On Tue, Oct 11, 2022 at 4:48 AM AlexandreRouma ***@***.***>
wrote:

> As I've already responded on march 18, there is no such thing as direct
> sampling on the RSPs. it receives HF directly with no other setting
> necessary.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AlexandreRouma/SDRPlusPlus/discussions/694#discussioncomment-3847794>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYJICJMM64JIBFJ7B5DEASDWCUSXJANCNFSM5RBYEYGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Matt Gillard
C:6477749124
